plugins/foxglove/Map and 3D layout.json

```diff
@@ -0,0 +1,94 @@
+{
+  "configById": {
+    "3D!3pzn6f1": {
+      "cameraState": {
+        "perspective": true,
+        "distance": 58.726522473994336,
+        "phi": 45.275535940625744,
+        "thetaOffset": 3.6088447450868943,
+        "targetOffset": [
+          2.8791262986708466,
+          -1.3958853068608803,
+          3.334144783479148e-16
+        ],
+        "target": [
+          0,
+          0,
+          0
+        ],
+        "targetOrientation": [
+          0,
+          0,
+          0,
+          1
+        ],
+        "fovy": 45,
+        "near": 0.5,
+        "far": 5000
+      },
+      "followMode": "follow-pose",
+      "scene": {},
+      "transforms": {},
+      "topics": {
+        "/atos/object_1/object_monitor": {
+          "visible": true,
+          "type": "arrow"
+        },
+        "/atos/object_1/path": {
+          "visible": true
+        },
+        "/atos/object_2/path": {
+          "visible": true
+        },
+        "/atos/object_3/path": {
+          "visible": true
+        },
+        "/atos/object_4/path": {
+          "visible": true
+        },
+        "/atos/object_5/path": {
+          "visible": true
+        },
+        "/atos/object_6/path": {
+          "visible": true
+        }
```

Comment:
Would be nice to add the same colors that Samuel added for rviz paths and position arrows, to be able to separate the objects.